src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -212,6 +223,14 @@ public boolean isFilesonly()
         return doFilesonly;
     }
 
+    public Long getFixedEntryModificationTime() {
+        return fixedEntryModificationTime;
+    }
+
+    public void setFixedEntryModificationTime(Long fixedEntryModificationTime) {
```

Comment:
Actually would it be even better if it works the same way as forced file/dir modes? I mean they are logically related - forced file/dir modes are options to override the file permissions. This is option to override the mast modified date.